packages/core/plugins/core-plugins-server-internal/src/plugins_config.test.ts

```diff
@@ -31,4 +31,17 @@ describe('PluginsConfig', () => {
     const config = new PluginsConfig(rawConfig, env);
     expect(config.additionalPluginPaths).toEqual(['some-path', 'another-path']);
   });
+
+  it('retrieves shouldEnableAllPlugins', () => {
+    const env = Env.createDefault(REPO_ROOT, getEnvOptions({ cliArgs: { dev: true } }));
+    const rawConfig: any = {
+      initialize: true,
+      paths: ['some-path', 'another-path'],
+      __internal__: {
+        enableAllPlugins: true,
+      },
```

Comment:
I would just have gone with `plugins.forceEnableAll` without the `__internal__` shenanigans (if it's not documented it's not supported), but that works too